src/commonMain/kotlin/komem/litmus/Utils.kt

```diff
@@ -1,9 +1,11 @@
+package komem.litmus
+
 import platform.posix._SC_NPROCESSORS_ONLN
 import platform.posix.sysconf
 
 fun List<List<String>>.tableFormat(hasHeader: Boolean = false): String {
     val columnCount = maxOf { it.size }
-    val columnSizes = (0 until columnCount).map { i ->
+    val columnSizes = (0..<columnCount).map { i ->
```

Comment:
please add spaces

Reply:
the autoformatter explicitly removes these spaces

Reply:
Then disable this behavior in the IDEA settings ))

Reply:
Ok, it looks like official styleguide is actually recommends to remove spaces:
https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace

But I don't know, looks weird for me (harder to read), harder to refactor (in case when you decide to replace it to e.g. `downTo`), inconsistent with the rules for other binary expressions.